apiary.apib

```diff
@@ -2430,6 +2430,57 @@ Create a quotation.
             + type: `quotation` (string)
             + id: `e4bf74ca-e900-471f-84b1-276e5d3afae4` (string)
 
+### quotations.send [POST /quotations.send]
+
+Send a quotation.
+
++ Request (application/json)
+
+    + Attributes (object)
+        + quotations: `023a4609-eda4-006c-8d2c-314539ec5d85`, `b42635b7-ccd3-0bfc-9628-e90940694845` (array[string], required) - list of quotation ids you wish to send. These quotations need to come from the same deal.
+        + from (object, optional)
+            + sender (object, required)
+                + type (enum, required)
+                    + Members
+                        + user
+                        + department
+                + id: `2659dc4d-444b-4ced-b51c-b87591f604d7` (string, required)
+            + email_address: `info@teamleader.eu` (string, required)
+        + recipients (object, required)
+            + to (array)
```

Comment:
Q: Shouldn't this be marked as `required`?

Reply:
You're right @tiagobastos 

```json
{
	"errors": [
		{
			"code": 0,
			"title": "to must be present",
			"status": 400,
			"meta": {
				"field": "to"
			}
		},
```

Fixed: https://github.com/teamleadercrm/api/pull/654/commits/0c74650f91449b389e55c1acfbf1b0620922af4d